packages/amazon-bedrock/src/bedrock-embedding-model.ts

```diff
@@ -27,7 +27,7 @@ type DoEmbedResponse = Awaited<ReturnType<EmbeddingModelV2<string>['doEmbed']>>;
 export class BedrockEmbeddingModel implements EmbeddingModelV2<string> {
   readonly specificationVersion = 'v2';
   readonly provider = 'amazon-bedrock';
-  readonly maxEmbeddingsPerCall = undefined;
+  readonly maxEmbeddingsPerCall = Infinity;
```

Comment:
this seems wrong - is there any max we can find out from their docs?

Reply:
aha looks like it's limited to just 1, and we do parallel requests good catch - will try to infer from code next time